cogs/customreactions.py

```diff
@@ -0,0 +1,1111 @@
+                        embed = discord.Embed(title=title)
+                        embed.set_footer(text="Modified by {}.".format(user),
+                                         icon_url=user.avatar_url)
+                        await message.edit(embed=embed)
+                        conn.close()
+                        time.sleep(5)
+                    else:
+                        t = (1, custom_react_id)
+                        c.execute(
+                            'UPDATE CustomReactions SET DM = ? WHERE CustomReactionID = ?',
+                            t)
+                        conn.commit()
+                        self.rebuild_reaction_list()
+                        self.rebuild_proposal_list()
+                        if proposals:
+                            title = "Option successfully modified! Returning to list of current reaction proposals..."
+                        else:
+                            title = "Option successfully modified! Returning to list of current reactions..."
+                        embed = discord.Embed(title=title)
+                        embed.set_footer(text="Modified by {}.".format(user),
+                                         icon_url=user.avatar_url)
+                        await message.edit(embed=embed)
+                        conn.close()
+                        time.sleep(5)
+                elif reaction.emoji == "⏹":
+                    await leave(message)
+                    return
+
+            if reaction.emoji == "✅":
+                t = (0, custom_react_id)
+                c.execute(
+                    'UPDATE CustomReactions SET Proposal = ? WHERE CustomReactionID = ?',
+                    t)
+                conn.commit()
+                self.rebuild_reaction_list()
+                self.rebuild_proposal_list()
+                title = "Custom reaction proposal successfully approved! " \
+                        "Returning to list of current reaction proposals..."
+                footer = "Aproved by {}.".format(user)
+                embed = discord.Embed(title=title)
+                embed.set_footer(text=footer, icon_url=user.avatar_url)
+                await message.edit(embed=embed)
+                conn.close()
+                time.sleep(5)
+
+            if reaction.emoji == "🚮" or reaction.emoji == "❌":
+                t = (custom_react_id, )
+                c.execute(
+                    'DELETE FROM CustomReactions WHERE CustomReactionID = ?',
+                    t)
+                conn.commit()
+                if proposals:
+                    title = "Custom reaction proposal successfully rejected! " \
+                            "Returning to list of current reaction proposals..."
+                    footer = "Rejected by {}.".format(user)
+                else:
+                    title = "Custom reaction successfully deleted! Returning to list of current reactions..."
+                    footer = "Deleted by {}.".format(user)
```

Comment:
duplicated string literal – use variable and format as needed:

```python
PROPOSAL_RESULT = "Custom reaction proposal successfully {}! " \
                        "Returning to list of current reaction proposals..."
```

Reply:
See https://github.com/idoneam/Canary/pull/273#issuecomment-656869764